lib/image-stream/handlers.js

```diff
@@ -76,6 +77,7 @@ module.exports = {
   'application/gzip': (file, options) => {
     return gzipUncompressedSize.fromFileAsync(file).then((uncompressedSize) => {
       return Bluebird.props({
+        path: file,
```

Comment:
What would the `path` be set to in the case where we're doing a streaming download?